src/data/h1/tags/weapon.yml

```diff
@@ -389,36 +389,61 @@ comments:
       en: ...
       fields:
         - name: flags
-          en: ...
+          en: Various flags that change the behavior of the given trigger.
           fields:
             - name: tracks fired projectile
               en: ...
             - name: random firing effects
-              en: ...
+              en: >
+                If false the firing effects in the firing effects block will be
+                picked in order. If true they will be picked randomly.
             - name: can fire with partial ammo
-              en: ...
+              en: >
+                The weapon will be able to fire as long as the magazine content is
+                above `0`.
             - name: does not repeat automatically
-              en: ...
+              en: >
+                Disables repeat fires while holding the trigger. If true you need
+                to release the trigger before it can fire again.
+                This breaks if the weapon is a charge weapon like the plasma pistol/
+                when the player is a client in multiplayer it will start charging
+                again as soon as it is fired. (This bug is only noticeable if
```

Comment:
This sentence feels grammatically off. Prefer to separate longer phrases with an "or" instead of a "/" as it's clearer. The transition to "...it will start charging" is rather abrubt as well. Suggestion:

"This breaks if the weapon is a charge weapon like the plasma pistol, or when the player is a client in multiplayer; it will start charging again as soon as it is fired."

Reply:
The / is an accident. That's supposed to be one sentence